examples/media/schema.json

```diff
@@ -47,61 +46,64 @@
           "post": {
             "description": "media can be any media object. All `encapsulating` media objects like album, playlist etc. will be flattend to root level of mediacollection. E.g. if a playlist contains track A and B, the tracks A and B will be added to the mediacollection. Changing an existing mediaCollectionObject would mean a modification of its `items` property content (e.g. adding or removing items from an existing collection)",
             "parameters": {
-              "resource": "media.collections",
-              "method": "post",
-              "required": [
-              "items"
-              ],
               "properties": {
                 "items": {
-                  "description": "media identifier",
+                  "isRequired": true,
```

Comment:
That kind of "required" is not clear enough because it's not obvious what properties are mandatory.

Reply:
Is required mean mandatory for the described operation

e.g. IsRequired on POST means 'mandatory for POST'

Reply:
How do you differentiate between POST on resource and POST on element? By the statemaent location in endpoints (i.e. resource section or element section)?

Reply:
The differentiation is done by the location of the `post` object. A resource has a `post`, and below elements, you also can have/find `post`.